indico/modules/users/util.py

```diff
@@ -441,3 +441,8 @@ def send_avatar(user):
     return send_file('avatar.png', BytesIO(user.picture), mimetype=metadata['content_type'],
                      inline=True, conditional=True, last_modified=parse_date(metadata['lastmod']),
                      cache_timeout=(86400*7))
+
+
+def get_avatar_from_name(first_name):
```

Comment:
```suggestion
def get_avatar_url_from_name(first_name):
```

that name is clearer IMO (don't forget to chance the places where it's called/imported as well ofc)

Reply:
Renamed the function